platform/Cargo.toml

```diff
@@ -12,6 +12,7 @@ readme.workspace = true
 
 [dependencies]
 async-trait = "0.1.63"
+async-std = "1.12.0"
```

Comment:
It was to get an implementation for this:
```rror[E0599]: no method named `next` found for struct `mongodb::Cursor` in the current scope
  --> platform/src/mongodb/analytics.rs:77:22
   |
77 |         match cursor.next().await {
   |                      ^^^^ method not found in `Cursor<Document>`
```

Since you pointed the std crate out, I was able to find a replacement that seems to work.  I'm using use futures_util::StreamExt; instead now.